analyzers/src/SonarAnalyzer.Common/Helpers/SonarAnalysisContext.cs

```diff
@@ -161,6 +161,20 @@ internal void RegisterSyntaxNodeAction<TLanguageKindEnum>(Action<SyntaxNodeAnaly
         internal ProjectConfigReader ProjectConfiguration(AnalyzerOptions options) =>
             ProjectConfiguration(context.TryGetValue, options);
 
+        internal static bool IsAnalysisScopeMatching(Compilation compilation, bool isTestProject, IEnumerable<DiagnosticDescriptor> diagnostics)
+        {
+            if (compilation == null)
```

Comment:
I wanted to remove this but the `LineContinuationTest` VB test started failing (no issue raised)